app/routes/bestuurseenheid.js

```diff
@@ -3,40 +3,22 @@ import { inject as service } from '@ember/service';
 
 export default class BestuurseenheidRoute extends Route {
   @service router;
-  breadCrumb = {};
+  @service store;
 
   async model(params, transition) {
     const bestuurseenheden = await this.store.query('bestuurseenheid', {
-      'include': 'classificatie',
+      include: 'classificatie',
       'filter[:exact:naam]': params.bestuurseenheid_naam,
-      'filter[classificatie][:exact:label]': params.bestuurseenheid_classificatie_code_label
+      'filter[classificatie][:exact:label]':
+        params.bestuurseenheid_classificatie_code_label,
     });
     if (bestuurseenheden.length == 0) {
       return null;
-    }
-    else {
+    } else {
       transition.data.bestuurseenheidNaam = params.bestuurseenheid_naam;
-      transition.data.bestuurseenheidClassificatieLabel = params.bestuurseenheid_classificatie_code_label;
+      transition.data.bestuurseenheidClassificatieLabel =
```

Comment:
I think the transition.data lines can be removed since they don't seem to be used anymore? I probably forgot to delete it during the breadcrumbs refactor.